portfolio/src/main/webapp/WEB-INF/nuzen_fitbit_data.csv

```diff
@@ -0,0 +1,28 @@
+"2020-05-26 2:29AM","2020-05-26 9:58AM","362","87","22","449","47","247","68"
```

Comment:
Can you write briefly in your PR description how this data was collected (what steps were used), How can this be uploaded, May be a stretch goal would be to provide a UI to upload this file.

Documenting these things is useful for your own reference too, a few weeks or months down the line it will difficult to remember how you even got this data.

Reply:
Sounds good -- I'll fix the above comments and add some documentation into the PR.